roles/pbx/tasks/install.yml

```diff
@@ -0,0 +1,72 @@
+- name: "Set 'nodejs_install: True' and 'nodejs_enabled: True'"
+  set_fact:
+    nodejs_install: True
+    nodejs_enabled: True
+
+- name: NODEJS - run 'nodejs' role (attempt to install & enable Node.js)
+  include_role:
+    name: nodejs
+
+- name: FAIL (STOP THE INSTALL) IF 'nodejs_installed is undefined'
+  fail:
+    msg: "PBX install cannot proceed, as Node.js is not installed."
+  when: nodejs_installed is undefined
+
+- name: FAIL (STOP THE INSTALL) IF 'nodejs_version != "10.x"'
+  fail:
+    msg: "PBX install cannot proceed, as it currently requires Node.js 10.x, whereas nodejs_version is set to {{ nodejs_version }}.  Please check the value of nodejs_version in /opt/iiab/iiab/vars/default_vars.yml, /etc/iiab/local_vars.yml, /opt/iiab/iiab/roles/nodejs, etc."
+  when: nodejs_version != "10.x"
```

Comment:
If you are touching pbx perhaps it would be a good time to address #2210 and correct this test.

Reply:
I don't know how to solve #2210 so am keeping that separate.

As important as it is.

@tim-moody may have ideas off of PR #2239 possibly.